components/script/dom/htmlformelement.rs

```diff
@@ -1689,6 +1690,19 @@ pub trait FormControl: DomObject {
         }
     }
 
+    // https://html.spec.whatwg.org/multipage/#autofill-expectation-mantle
+    fn autofill_mantle(&self) -> AutofillMantle {
```

Comment:
I think it would be clearer if this was a function in `autofill` taking the element as argument. Also I think it could return a boolean and be renamed `is_anchor`, so that the enum can instead be used to store data.